src/Scripting/VisualBasic/VisualBasicScriptCompiler.vb

```diff
@@ -86,7 +86,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Scripting
                     xmlReferenceResolver:=Nothing, ' don't support XML file references in interactive (permissions & doc comment includes)
                     sourceReferenceResolver:=SourceFileResolver.Default,
                     metadataReferenceResolver:=script.Options.MetadataResolver,
-                    assemblyIdentityComparer:=DesktopAssemblyIdentityComparer.Default),
+                    assemblyIdentityComparer:=DesktopAssemblyIdentityComparer.Default).
+                    WithIgnoreCorLibraryDuplicatedTypes(True),
```

Comment:
VB scripting didn't use that option previously (only the expression compiler). I was a bug.
In C#, both scripting and EE use `BinderFlags.IgnoreCorLibraryDuplicatedTypes`.

@tmat may want to comment.